blog/_posts/2025-07-25-aks-lts-announcement.md

```diff
@@ -0,0 +1,242 @@
+- **Undrainable node behavior:** Configure how AKS handles nodes that cannot be drained, ensuring predictable upgrade outcomes
+- **OS Security Patch channel:** Automate operating system security updates through configurable patch channels
+- **Node Image channel:** Keep node images updated with the latest security patches and OS improvements
+- **Planned maintenance windows:** Schedule upgrades during off-peak hours to minimize business impact
+
+For comprehensive guidance on all available upgrade strategies, see the [AKS cluster upgrade documentation](https://learn.microsoft.com/azure/aks/upgrade-cluster). These production-tested upgrade mechanisms ensure smooth transitions between LTS versions while maintaining workload availability.
+
+## Transitioning Between Support Models
+
+**Community to LTS Transition:**
+Moving from community to LTS support is straightforward—simply upgrade your cluster to the Premium tier. Your current Kubernetes version immediately receives extended 24-month support coverage without requiring a cluster upgrade or migration.
+
+**LTS to Community Transition:**
+Transitioning from LTS to community support requires more planning, especially if you're near the end of your 24-month LTS window. Key considerations include:
+
+- **Version gap assessment:** After 24 months on LTS, you may need to upgrade across multiple Kubernetes versions to reach a currently supported community version
+- **Multi-hop upgrades:** While Kubernetes upstream focuses on n-1 to n upgrades, AKS provides support for multi-version upgrades with [upgrade path guidance](https://learn.microsoft.com/azure/aks/upgrade-cluster#upgrade-an-aks-cluster) and testing recommendations for larger version jumps
+- **Shared responsibility:** You're responsible for workload compatibility testing across version gaps, while AKS ensures the upgrade path is technically viable and provides breaking change documentation
+- **Planning window:** Begin transition planning 6-12 months before your LTS support expires to allow adequate time for testing and validation
+
+**Best Practice:** Consider your long-term strategy when initially choosing LTS to avoid complex transitions later.
+
+## Looking Ahead
+
+AKS LTS represents our commitment to supporting enterprises at every stage of their Kubernetes journey. While LTS focuses on stability, we continue to innovate rapidly in standard AKS, ensuring you have access to the latest Kubernetes capabilities when you need them.
+
+Over the coming months, we'll be expanding AKS LTS capabilities based on your feedback, including:
+
+- **Istio support for LTS:** Bringing comprehensive LTS support for Istio service mesh to provide enterprise-grade stability for your microservices architecture
+- **KMS V2 support for LTS:** Enhanced Key Management Service V2 support for AKS LTS coming by early CY26, providing improved encryption key management capabilities for enterprise security requirements
```

Comment:
```suggestion
- [KMS v2 for encryption of data at rest in etcd](https://learn.microsoft.com/en-us/azure/aks/use-kms-etcd-encryption) is currently being redesigned with planned enhancements on versionless key inputs for customer managed keys and default encryption using platform-managed keys. LTS is planned tentatively in CY2026H1 once these enhancements are released and generally available.
```

Reply:
modified to tent date already for KMS V2.